src/web/routes/application/check/create-request-body.js

```diff
@@ -0,0 +1,33 @@
+const { toDateString } = require('../common/formatters')
```

Comment:
There is no new code here - just moved into a new file to reduce the size of `post.test.js`